tests/test_aea.py

```diff
@@ -306,12 +304,12 @@ def test_initialize_aea_programmatically(self):
         assert dummy_behaviour is not None
         assert dummy_behaviour.nb_act_called > 0
 
-        dummy_task_name = "dummy"
-        dummy_task = self.aea.resources.task_registry.fetch(
-            (dummy_skill_id, dummy_task_name)
-        )
-        assert dummy_task is not None
-        assert dummy_task.nb_execute_called > 0
+        # TODO the previous code caused an error:
+        #      _pickle.PicklingError: Can't pickle <class 'tasks.DummyTask'>: import of module 'tasks' failed
```

Comment:
I mentioned this before. I can see it as a problem in all the cases where the `Task` object contains data that are non-pickable (e.g. `skill_context`), or that they are quite cumbersome in terms of memory (Gigabytes? probably not a problem in the short term).

Reply:
Ok, add to docs imo